doc/lua_api.txt

```diff
@@ -6201,6 +6198,21 @@ Used by `minetest.hud_add`. Returned by `minetest.hud_get`.
         size = { x=100, y=100 },
     --  ^ Size of element in pixels
     }
+    
+Player HUD Flags
```

Comment:
Seems out of place here, it is a table of parameters for an API, not a definition table.